mqtt/mqtt-edgehub/src/auth/authorization/edgehub.rs

```diff
@@ -263,15 +263,15 @@ where
     fn update(&mut self, update: Box<dyn Any>) -> Result<(), Self::Error> {
         match update.downcast::<AuthorizerUpdate>() {
             Ok(update) => {
-                debug!("authorizer update received. Identities: {:?}", update);
-
                 // update identities cache
                 self.identities_cache = update
                     .0
                     .into_iter()
                     .map(|id| (id.identity().into(), id))
                     .collect();
 
+                info!("edgehub authorizer has been updated.");
+
                 // signal that authorizer has been initialized
                 if let Some(mut broker_ready) = self.broker_ready.take() {
                     broker_ready.send(BrokerReadyEvent::AuthorizerReady);
```

Comment:
Could you please add debug messages, that signal has been sent to broker ready component? It will help to investigate issues if broker cannot serve external clients too long